libraries/Microsoft.Bot.Builder.Dialogs/Prompts/OAuthPrompt.cs

```diff
@@ -59,6 +61,7 @@ public class OAuthPrompt : Dialog
         private const string PersistedOptions = "options";
         private const string PersistedState = "state";
         private const string PersistedExpires = "expires";
+        private const string PersistedCaller = "caller";

```

Comment:
Namespace?

Reply:
Does it make sense, considering all the other stuff is not namespaced? and we cannot change the others because we would break backward compat for folks that have persisted state in their storage.

Reply:
I agree on not changing the others older consts because of the broken state, but we should be more cautious going forward and namespace any keys that will be persisted in storage. 

We could guarantee when we went GA that the consts wouldn't collide with any customer values, but we don't have that guarantee now.

Reply:
Sounds good.